archivy/templates/dataobjs/show.html

```diff
@@ -107,6 +114,46 @@ <h2 id="post-title">{{ dataobj['title'] }}</h2>
           oldContent = document.getElementById("content"), editorDiv = document.querySelector(".EasyMDEContainer");
           editorDiv.classList.add("hidden");
           document.getElementById("post-title").scrollIntoView(true);
+
+          /* Screenreader mode checkbox */
+          function toggleScreenReaderMode(cbox, editor)
+          {
+            return function ()
```

Comment:
Do we need an inner function here, or could we just directly put the if/else?

(Just curious, I might be making a mistake :smile: )

Reply:
Yes, we need the inner function in order to pass the editor object to the function. I think we could remove the cbox parameter and use "this" to reference it from the inside, but I think it is clearer this way.